### ISSUE TITLE
any possibility to release pre-modded x86 builds as well (and not only arm)?

### ISSUE THREAD

**Issue by wokawoka** [sergiocastell/spotify-dogfood#21]
it would be very useful for people running android on x86 platforms ...
thanks for this great project!

**Comment by sergiocastell** [comment]
No, since arm versions work right away on x86 (?)

**Comment by wokawoka** [comment]
Not really, since spotify-dogfood after a while always crashes while multitasking on my x86 tablet (for example when browsing and listening to music at the same time).
This doesn't happen when using spotify installed from google play.
But thanks anyway

**Comment by sergiocastell** [comment]
Interesting. I'll have a look then.

**Comment by wokawoka** [comment]
thank you Sergio. 
Please just keep in mind that the it was just an assumption of mine: I thought that the problem with the crashes (or process terminations) while multitasking could have been in the arm emulation layer (as, on my x86 tablet I'm not experiencing the same behaviour with regular spotify downloaded from Google play, which I assume is compiled for x86).
But apparently it looks like that other users are experiencing the same behaviour (maybe on arm?) https://forum.xda-developers.com/showpost.php?p=74715916&postcount=541 , so I believe that the problem could maybe be linked to something else.
Thanks again

**Comment by budianto7** [comment]
true it doesn't want to install on my zenfone 2 intel soc (ze551ml). it need its own x86.

**Comment by tletum13** [comment]
Not really, since spotify-dogfood after a while always crashes while multitasking on my x86 tablet (for example when browsing and listening to music at the same time).
This doesn't happen when using spotify installed from google play.
But thanks anyway

**Comment by rmj35** [comment]
I doesn't install in my zenfone 2 too... 😭
